Resources/Prototypes/_Backmen/Catalog/Fills/Items/toolboxes/toolboxes.yml

```
@@ -0,0 +1,16 @@
+  components:
+  - type: StorageFill
+    contents:
+    - id: Crowbar
+    - id: Wrench
+    - id: Screwdriver
+    - id: Wirecutter
+    - id: Welder
+    - id: Multitool
+    - id: ClothingHandsGlovesCombat
+    - id: ClothingBalaclavaSwat
+    
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)